README.md

```diff
@@ -136,3 +136,28 @@ If you do not have Python 3.10 installed, you can use root-less podman with a Py
 ```bash
 podman run --rm --tty --volume "$(pwd):/mnt:z" --workdir /mnt docker.io/library/python:3.10 bash -c 'pip install nox ; nox -s pip-compile'
 ```
+
+## Creating release tags
+Creating tags that correspond with releases and release candidates in [`ansible/ansible`](https://github.com/ansible/ansible)
```

Comment:
```suggestion
When a tag is created in the [`ansible/ansible`](https://github.com/ansible/ansible) repository for a release or release candidate, a corresponding tag should be created in this `ansible-documentation` repository.
```

Reply:
It's probably a bit wordier but I think we need to explain a bit more clearly "when" to create tags.

Reply:
Agreed. This absolutely makes it more clear.